appveyor.yml

```diff
@@ -2,12 +2,6 @@
 # file, make changes to conda-forge.yml and/or recipe/meta.yaml, and run
 # "conda smithy rerender".
 
-
-# Post-conda-smithy customization: build on this OS until we can figure out
-# how to workaround a compiler bug introduced in Update 3.
-os:
-    - Visual Studio 2015 Update 2
```

Comment:
Not sure if that was the culprit for #17. If not You'll need to restore this.

Reply:
Same errors, so it seems that it is not. I will restore it

Reply:
Yep. I'll take a closer look tomorrow.

Reply:
Hi @ocefpaf, is there any workaround? Should I just update the build number to resolve my problem? I want to announce the release but we need this conda package first. 
Thanks !

Reply:
> Hi @ocefpaf, is there any workaround? Should I just update the build number to resolve my problem? I want to announce the release but we need this conda package first.
Thanks !

Not sure what you mean. How would bumping the build number would fix the compiler issue on Windows? You need someone with more knowledge of Windows to help here. I am not that person :grimacing:

Reply:
it does not seems to be a compiler issue (otherwise, we know how to manage it) but more conda installation problem like I said on #17